dreem/inference/eval.py

```
@@ -26,26 +26,33 @@ def run(cfg: DictConfig) -> dict[int, sio.Labels]:
+        _ = hparams.pop("Unnamed: 0", None)
+
+        if eval_cfg.set_hparams(hparams):
+            logger.info("Updated the following hparams to the following values")
+            logger.info(hparams)
     else:
-        checkpoint = eval_cfg.cfg.ckpt_path
+        hparams = {}
 
     logging.getLogger().setLevel(level=cfg.get("log_level", "INFO").upper())
 
     model = GTRRunner.load_from_checkpoint(checkpoint, strict=False)
```

Ruff (0.8.2)

Undefined name `checkpoint`

(F821)